gui/dig.lua

```diff
@@ -632,6 +691,7 @@ DigScreen = defclass(DigScreen, gui.ZScreen)
 DigScreen.ATTRS {
     focus_path = "dig",
     pass_pause = true,
+    initial_pause = false,
```

Comment:
you should probably leave this unspecified so the player can configure their preference via gui/control-panel